Cargo.toml

```diff
@@ -1,15 +1,21 @@
 [package]
-name = "rust-memtester"
+name = "memtest"
+description = "A library for detecting faulty memory"
 version = "0.1.0"
+authors = ["Brian Tsoi <brian.s.tsoi@gmail.com>"]
 edition = "2021"
+license = "MPL-2.0"
+homepage = "https://github.com/BrianShTsoi/memtest"
+repository = "https://github.com/BrianShTsoi/memtest"
+keywords = ["memory", "stability"]
+readme = "README.md"
 
 [dependencies]
-anyhow = "1.0"
-libc = "0.2"
-num_cpus = "1.0"
-rand = "0.8.4"
-region = "3.0.2"
-tracing = "0.1.40"
+anyhow = "1.0.69"
+libc = "0.2.158"
```

Comment:
Just a quick thought, but do you use this library on Windows at all? Should it be in `[dependencies.unix]`?